app/src/main/java/io/homeassistant/companion/android/vehicle/MainVehicleScreen.kt

```diff
@@ -121,6 +126,18 @@ class MainVehicleScreen(
                 }
             }
         }
+        lifecycleScope.launch {
+            favoriteEntites = allEntities.map {
```

Comment:
yea they take an extra second to load, i noticed that too 🤔

Reply:
I feel like they never show up bit this might also be helped/fixed by fixing the [loading icon](https://github.com/home-assistant/android/pull/3694#discussion_r1271137216) to work correctly

Reply:
now favorites will show once all entities are loaded